DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -834,3 +834,23 @@ function SetAllLampsState(bool type1, bool type2, bool type3, optional Vector lo
     }
 #endif
 }
+
+function MoveSmugglerElevator()
+{
+    local #var(prefix)DeusExMover elevator;
+    local bool street;
+
+    if (!#defined(vanilla)) return;
+
+    street = dxr.localURL == "02_NYC_STREET" || dxr.localURL == "04_NYC_STREET" || dxr.localURL == "08_NYC_STREET";
+
+    foreach AllActors(class'#var(prefix)DeusExMover', elevator, 'elevatorbutton') {
```

Comment:
Rather than hardcoding 'elevatorbutton' in here, you should just use your configured tag, since it will match the elevator itself, and in the possibility we end up using this on a different version of a map (using a different tag), it can be adjusted at spawn time

Reply:
What do you mean by "your configured tag"?

Reply:
Ah, you just mean 'elevatorbutton'.